lib/sqlalchemy/engine/interfaces.py

```diff
@@ -257,13 +257,13 @@ def get_columns(self, connection, table_name, schema=None, **kw):
           boolean
 
         * ``sequence`` -
-          a dictionary of the form
+          a dictionary of the form::
               {'name' : str, 'start' :int, 'increment': int, 'minvalue': int,
```

Comment:
Without ``::`` it was ``./lib/sqlalchemy/engine/interfaces.py:261:1: RST301 Unexpected indentation.`` see details in [report](https://jenkins.sqlalchemy.org/job/pep8/3417/consoleText)

I suppose now it will pass the test successfully.
Now it looks like this:
![Screenshot from 2020-06-26 01-03-33_](https://user-images.githubusercontent.com/5857672/85801311-4bc73c80-b74b-11ea-9e62-1997de0cd9c2.png)